content/en/docs/ambient/architecture/index.md

```diff
@@ -9,15 +9,63 @@ owner: istio/wg-networking-maintainers
 test: n/a
 ---
 
+## Ambient APIs
+
+### Labels
+
+You can use the following labels to enroll your namespace to ambient, enroll your resource to use a waypoint and specify, the capture scope of your waypoint.
+
+|  Name  | Feature Status | Scope | Description |
```

Comment:
The table format can be improved, suggest to use: http://markdowntable.com/ to format it.

Reply:
I tried to use it, but i think it is harder to view or make change in the source view.  BTW, the table renders fine in preliminary, which is what users would see.